<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>变量数</string>
    </property>
   </widget>
   <widget class="QComboBox" name="number_of_index">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>求解式</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>188</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_condition_number1">
         <property name="text">
          <string>变量名1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_condition_number2">
         <property name="text">
          <string>变量名2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_condition_number3">
         <property name="text">
          <string>变量名3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_condition_number4">
         <property name="text">
          <string>变量名4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLineEdit" name="condition_name_1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="condition_name_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="condition_name_3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="condition_name_4"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Activicate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>生成！</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="result">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>421</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>202</width>
      <height>104</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="condition1_label">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>条件1</string>
           </property>
           <property name="openExternalLinks">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="condition2_label">
           <property name="text">
            <string>条件2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="condition3_label">
           <property name="text">
            <string>条件3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="condition4_label">
           <property name="text">
            <string>条件4</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="condition_line1"/>
         </item>
         <item>
          <widget class="QLineEdit" name="condition_line2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="condition_line3"/>
         </item>
         <item>
          <widget class="QLineEdit" name="condition_line4"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="zero_1">
         <property name="text">
          <string>=0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="zero_2">
         <property name="text">
          <string>=0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="zero_3">
         <property name="text">
          <string>=0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="zero_4">
         <property name="text">
          <string>=0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
